dev/ci/ci-sf.sh

```diff
@@ -3,7 +3,11 @@
 ci_dir="$(dirname "$0")"
 . "${ci_dir}/ci-common.sh"
 
+CIRCLE_SF_TOKEN=00127070c10f5f09574b050e4f08e924764680d2
 mkdir -p "${CI_BUILD_DIR}" && cd "${CI_BUILD_DIR}" || exit 1
+sf_lf_CI_TARURL=`curl https://circleci.com/api/v1.1/project/gh/DeepSpec/sfdev/latest/artifacts?circle-token=${CIRCLE_SF_TOKEN} | jq '.[].url' | grep \/lf.tgz`
```

Comment:
```suggestion
sf_lf_CI_TARURL=`curl https://circleci.com/api/v1.1/project/gh/DeepSpec/sfdev/latest/artifacts?circle-token=${CIRCLE_SF_TOKEN} | jq '.[] | select (.path == "lf.tgz") | .url'`
```
and so on for the others